sre-recipes/sandboxctl

```diff
@@ -119,6 +119,19 @@ def loadgen(traffic_pattern):
     print(f'Loadgenerator deployed using {traffic_pattern} pattern')
     Recipe._auth_cluster('APP')
 
+@cli.command()
+def create():
+    """Create a new sandbox"""
+    os.chdir(os.path.abspath(sys.path[0]))
+    create_command = "../terraform/install.sh"
```

Comment:
will this work if I run sandboxctl from any random directory in cloud shell - assuming that sandboxctl is on the $PATH?

Reply:
Yes, the line above this makes changes the working dir to be the location of the `sandboxctl` script, so it can be called from anywhere. We will need to make changes if we move the sandboxctl fine though